scripts/setup/setup-certbot

```diff
@@ -84,7 +93,11 @@ chmod a+x "$CERTBOT_PATH"
 # to agree to the Let's Encrypt Subscriber Agreement (aka ToS).
 # Passing --force-interactive suppresses a warning, but also brings up
 # an annoying prompt we stifle with --no-eff-email.
-"$CERTBOT_PATH" certonly "${method_args[@]}" -d "$DOMAIN" -m "$EMAIL" --force-interactive --no-eff-email
+"$CERTBOT_PATH" certonly "${method_args[@]}" \
+                -d "$DOMAIN" -m "$EMAIL" \
+                "$agree_tos" --force-renewal \
```

Comment:
Can you test that the `$agree_tos` argument works if it isn't set?  I'm worried about the shell treating processing the "" as an empty-string argument.

Reply:
(I'm pretty sure it's OK, but it's the core installer flow, so important to test.  Or we could just do something guaranteed to be safe against that, e.g. having 

`CERTBOT_BASE_ARGS="--force-interactive --no-eff-email $agree_tos"
)

Reply:
I ended up having a convenient opportunity to test this, and it's actually broken, so I'm glad I tested this.

Reply:
I think just removing the quotes around ``"$agree_tos"` would be correct here.

Reply:
Yep, I see what you mean. Fixed!